.github/scripts/generate_changelog.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+echo "Step 1: Preparing revision folder...."
+
+# The revision folder includes"
+# 1) OpenAPI spec files that are currently in the repository are copied to the revision folder
+# 2) changelog Metadata file that is generated via foascli
+# 3) exemptions.yaml file that is downloaded from S3 
+mkdir changelog/revision
+cp openapi/v2/openapi-*.json changelog/revision/
```

Comment:
Do we need to cleanup folder when running locally? May be worth to use `-Rf` otherwise

Reply:
I am cleaning this up as part of the GH action instead of doing it here since I want to upload base and revision folders to help to debug issue if needed before deleting them

Reply:
This is the piece of code that I am referring to in "release-changelog.yml"
```yaml
        - name: Upload revision and base folders for debugging
          uses: actions/upload-artifact@834a144ee995460fba8ed112a2fc961b36a5ec5a
          with:
            name: base-revision-${{ inputs.env }}
            retention-days: 1
            path: |
              changelog/base
              changelog/revision
        - name: Clean up base and revision folders
          run: |
            rm -rf changelog/base
            rm -rf changelog/revision
```

Reply:
SGMT

Reply:
I think as side goal I would consider running those in github actions and locally. 


As side note: [Repository can have some contributing guide[ with repo structure and what scripts can be run locally and what are github actions bound.

Reply:
If this is done after I am gone, all the Postman scripts are able to be run locally assuming you have some environment variables set. I'll update the README so it is clear